src/lib/i18n/locales/fr-FR/translation.json

```diff
@@ -562,17 +562,17 @@
 	"Pull a model from Ollama.com": "Télécharger un modèle depuis Ollama.com",
 	"Query Params": "Paramètres de requête",
 	"RAG Template": "Modèle RAG",
-	"Rating": "",
-	"Re-rank models by topic similarity": "",
+	"Rating": "Note",
+	"Re-rank models by topic similarity": "Reclasser les modèles par similarité de sujet",
 	"Read Aloud": "Lire à haute voix",
 	"Record voice": "Enregistrer la voix",
 	"Redirecting you to OpenWebUI Community": "Redirection vers la communauté OpenWebUI",
 	"Refer to yourself as \"User\" (e.g., \"User is learning Spanish\")": "Désignez-vous comme « Utilisateur » (par ex. « L'utilisateur apprend l'espagnol »)",
-	"References from": "",
+	"References from": "Références de",
 	"Refused when it shouldn't have": "Refusé alors qu'il n'aurait pas dû l'être",
 	"Regenerate": "Regénérer",
-	"Release Notes": "Notes de publication",
-	"Relevance": "",
+	"Release Notes": "Notes de mise à jour",
```

Comment:
"Notes de publication" is not incorrect, but "Notes de mise à jour" (literally "Update notes") is a bit more natural, I suppose